tools/nixd/test/basic-diagnostic.md

```diff
@@ -4,33 +4,32 @@
 
 ```json
 {
-   "jsonrpc":"2.0",
-   "id":0,
-   "method":"initialize",
-   "params":{
-      "processId":123,
-      "rootPath":"",
-      "capabilities":{
-      },
-      "trace":"off"
-   }
+  "jsonrpc": "2.0",
```

Comment:
as noted in `docs/dev.md`, I think you should *NOT* touch these regression tests, because

> Regression tests are written in markdown

This our testing framework that checks the `nixd` binary is responding correctly.